assets/css/static-main.scss

```diff
@@ -0,0 +1,294 @@
+---
+---
+@import 'styles.scss';
+
+.api-pattern {
+    background-image: url(../img/iconsbcda.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: top right;
+    width: 100%;
+}
+
+.blueberry-lime-gradient-background {
+background: linear-gradient(30deg, #339392, #339392 73%, #62C5C4);
+}
+
+.solid-container {
+    width: 60%;
+    background-color: #339392;
+}
+
+article h2{
+  display: inline-block;
+  position: relative;
+}
+
+article h2::after {
+    content: "";
+    position: absolute;
+    bottom: -8px;
+    left: 0;
+    height: 4px;
+    width: 100%;
+    background: linear-gradient(30deg, #087BAE 10%, #34EFA4 90%);
+}
+.subnav .blueberry-lime-link:active:before,
+.subnav .blueberry-lime-link:focus:before,
+.subnav .blueberry-lime-link:hover:before {
+    background: linear-gradient(30deg, #34EFA4 10%, #087BAE 90%)
+}
+
+.logo-link-container{
+  color: #0071BC
+}
+
+.ds-c-desktop-nav .desktop-nav-items a, .mobile-nav-items a{
+  color: #0071BC
+}
+
+.bcda_callout {
+    background-color: #F5F5F5;
+    padding: 20px;
+}
+
+.no_list{
+  list-style: none;
+}
+
+.button p{
+  text-align: center;
+  width:100%;
+  margin: auto;
+}
+
+.button p a{
+  color:white;
+  text-decoration: none;
+  width:60%;
+  border-radius:3px;
+  padding: 10px;
+  background-color: #0071BC;
+  display:block;
+  margin:auto;
+}
+
+#google_logo {
+    max-height: 60px;
+    float: right;
+    margin-top: -15px;
+    padding-bottom: 25px;
+  }
+
+.ds-text--lead {
+    font-size: 24px;
+}
+
+@media only screen and (max-width: 1090px){
+  .api-pattern {
+    background:none;
+    background-color: #339392;
+  }
+  .solid-container {
+      width: 100%;
+  }
+
+}
+
+@media only screen and (min-width: 1960px){
+    .solid-container {
+        width: 80%;
+    }
+    .blueberry-lime-gradient-background {
+        background: linear-gradient(30deg, #339392, #339392 83%, #62C5C4);
+    }
+
+}
+
+.topdrop {
+    display: inline-block;
+    margin-left: -10px;
+}
+
+.dropdown {
+    position: relative;
+    display: inline-block;
+    margin-right:20px;
+}
+
+/*
+.dropdown-content {
+    display: none;
+    position: absolute;
+    width:200%;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+    appearance: none;
+    background-color: #339392;
+    border-radius: 3px;
+    cursor: pointer;
+    font-family: "Open Sans", Helvetica, sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 1.3;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    transition: all .3s ease
+}
+
+.dropdown-content a {
+    text-decoration: none;
+    display: block;
+}
+*/
```

Comment:
will this be added later? would you mind commenting out why this is commented out?